whichone/static/css/main.css

```diff
@@ -51,6 +51,16 @@ textarea {
 
 /* ---------------------------------- */
 
+/* Cursors */
+
+.cursor-default {
+    cursor: default;
+}
+
+.cursor-pointer {
```

Comment:
Generally preferable for classes to describe the state of the DOM element (i.e. in this case I guess its an enabled button?)

So you'd have a single ruleset for a disable button.
```
button:disabled {
  /* all styles you want to apply to disabled buttons */
}
```